docs/usage.rst

```diff
@@ -1,6 +1,7 @@
 How to Use Django Phone Verify?
 =====================================
 
+``django-phone-verify`` provides few APIs which can be added to your list of API endpoints. It's functionality can be extended based on your usecase.
```

Comment:
I think it will be better to add this as a note under `Usage` section. Thoughts?

Reply:
Yeah, that'd look better. Agree with you!